client/apps/edit2/components/content/sections/images/components/controls.jsx

```diff
@@ -93,43 +120,43 @@ export class ImagesControls extends Component {
 
   render () {
     const {
-      articleLayout,
+      article,
       isHero,
-      section,
+      onChangeSectionAction,
+      editSection,
       setProgress
     } = this.props
 
     const inputsAreDisabled = this.inputsAreDisabled()
+    const section = isHero ? article.hero_section : editSection
 
     return (
         <SectionControls
-          section={section}
-          articleLayout={articleLayout}
-          sectionLayouts={!isHero}
+          showLayouts={!isHero}
           isHero={isHero}
-          disabledAlert={this.fillwidthAlert}
+          disabledAlert={this.fillWidthAlert}
         >
-          <div onClick={inputsAreDisabled ? this.fillwidthAlert : undefined}>
+          <div onClick={inputsAreDisabled ? this.fillWidthAlert : undefined}>
```

Comment:
I wonder if `{inputsAreDisabled && this.fillWidthAlert}` would work

Reply:
I get a console error with that, for functions it wants undefined to be explicitly declared.